.github/ISSUE_TEMPLATE/report-a-bug.md

```diff
@@ -0,0 +1,55 @@
+---
+name: Report a bug
+about: Have you found a bug or issue? Create a bug report for this SDK
+title: ''
+labels: bug report
```

Comment:
I just created this new label.

Reply:
every repo has a red "bug" label by default, AFAIK. Wouldn't that be easier to implement across repos?

Reply:
I like the "bug report" label to differentiate between a bug that's been verified, and a bug report. But if we want to discuss that further let's do so with the team as this comes from the templates that will be applied across all our repos.